### ISSUE TITLE
`mailto:` redirects still not working (?)

### ISSUE THREAD

**Issue by ndavd** [vikejs/vike#1380]
### Description

https://github.com/vikejs/vike/issues/1245
```
Error: [vike@0.4.151][Bug] You stumbled upon a bug in Vike's source code. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error; a maintainer will fix the bug (usually under 24 hours).
    at parseUrl (file:///(...)/node_modules/vike/dist/esm/utils/parseUrl.js:37:5)
    at removeUrlOrigin (file:///(...)/node_modules/vike/dist/esm/utils/parseUrl-extras.js:93:72)
    at file:///(...)/node_modules/vike/dist/esm/node/runtime/renderPage.js:344:50
    at modifyUrlPathname (file:///(...)/node_modules/vike/dist/esm/utils/parseUrl-extras.js:84:30)
    at getPermanentRedirect (file:///(...)/node_modules/vike/dist/esm/node/runtime/renderPage.js:340:21)
    at renderPageAndPrepare (file:///(...)/node_modules/vike/dist/esm/node/runtime/renderPage.js:97:41)
    at file:///(...)/node_modules/vike/dist/esm/node/plugin/shared/getHttpRequestAsyncStore.js:68:35
    at renderPage (file:///(...)/node_modules/vike/dist/esm/node/runtime/renderPage.js:46:50)
    at file:///(...)/server/index.ts:48:29
```

**Comment by brillout** [comment]
Insert the following before the line `37` of `file:///(...)/node_modules/vike/dist/esm/utils/parseUrl.js` and report back.
 
```diff
 function parseUrl(url, baseServer) {
+  console.log('url', url);
   assert(isParsable(url));
```

**Comment by ndavd** [comment]
```
12:16:43 PM [vike][request(1)] HTTP response / 200
12:16:48 PM [vike][request(2)] HTTP request: /mail
url /mail
url /mail
url /mail
url /mail
url /mail
url mailto:foo@bar.com
```

**Comment by brillout** [comment]
FYI @usk94.

**Comment by brillout** [comment]
Pre-released as `0.4.151-commit-19bc995`.

**Comment by ndavd** [comment]
Thanks
